BrainPortal/app/views/bourreaux/new.html.erb

```diff
@@ -0,0 +1,325 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+#
+-%>
+
+
+<% title 'Add New Server' %>
+
+<h2>Add New Server</h2>
+
+<%= error_messages_for @bourreau, :object_name => "server" %>
+
+<%= form_for @bourreau, :as => :bourreau, :url => { :action => "create" }, :datatype => "script" do |f| -%>
+
+  <div class="generalbox">
+
+	<% is_bourreau = @bourreau.is_a?(Bourreau) %>
```

Comment:
Simplify: the is_bourreau and is_portal checks are no longer necessary, this form will always be with is_bourreau set to true